docs: update creating vehicle and sensor model pages (#460)
autowarefoundation/autoware-documentation: docs/how-to-guides/integrating-autoware/creating-vehicle-and-sensor-model/creating-individual-params/index.md

# Creating individual params

## Introduction

> In cases where there is more than one same vehicle, each vehicle has the same sensor kit,
> but they may require different sensor calibrations, so
> [individual_params](https://github.com/autowarefoundation/autoware_individual_params) package allows you
> to define customized sensor calibrations for different vehicles while using the same launch
> vehicles or varying calibration requirements.

Reviewer: I believe this might be easier to understand:
```suggestion

The [individual_params](https://github.com/autowarefoundation/autoware_individual_params) package is used to define customized sensor calibrations for different vehicles. It lets you define customized sensor calibrations for different vehicles while using the same launch files.
```

Author: done :+1:

!!! Warning

    The "individual_params" package contains the calibration
    results for your sensor kit and overrides the default calibration results found in
    VEHICLE-ID_sensor_kit_description/config/ directory.

> We forked our autoware_individual_params repository
> at [creating autoware repositories](../../creating-your-autoware-meta-repository/creating-autoware-meta-repository.md) page step,
> (For example,
> we created [tutorial_vehicle_individual_params](https://github.com/leo-drive/tutorial_vehicle_individual_params) for our documentation vehicle at this step)
> please be sure `<YOUR-VEHICLE-NAME>_individual_params` repository is included in autoware like the directory below.
> Please create directory under the `config` directory with your vehicle id of your vehicle.
> (There will be one from the tutorial vehicle,
> so we will think 'tutorial_vehicle' as vehicle_id and vehicle_name)
> You need
> to create or copy [`imu_corrector.param.yaml`](https://github.com/autowarefoundation/autoware.universe/blob/main/sensing/imu_corrector/config/imu_corrector.param.yaml),
> [`sensor_kit_calibration.yaml`](https://github.com/autowarefoundation/sample_sensor_kit_launch/blob/main/sample_sensor_kit_description/config/sensor_kit_calibration.yaml) and [`sensors_calibration.yaml`](https://github.com/autowarefoundation/sample_sensor_kit_launch/blob/main/sample_sensor_kit_description/config/sensors_calibration.yaml) files
> under the `config/<YOUR-VEHICLE-ID>/<YOUR-VEHICLE-NAME>_sensor_kit_launch/` directory.
> Then, individual params will override these original files.
> 
> ```diff
> <YOUR-OWN-AUTOWARE-DIR>/
>   └─ src/
>        └─ param/
>             └─ <YOUR-VEHICLE-NAME>_individual_params/
>                  └─ individual_params/
>                     └─ config/
>                         ├─ default/
> +                       └─ <YOUR-VEHICLE-ID>/
> +                           └─ <YOUR-VEHICLE-NAME>_sensor_kit_launch/
> +                               ├─ imu_corrector.param.yaml
> +                               ├─ sensor_kit_calibration.yaml
> +                               └─ sensors_calibration.yaml
> ```
> 
> ??? note "sample folder structure for [`tutorial_vehicle_individual_params`](https://github.com/leo-drive/tutorial_vehicle_individual_params)"
> 
>     ```diff
>     <YOUR-OWN-AUTOWARE-DIR>/
>       └─ src/
>            └─ param/
>                 └─ tutorial_vehicle_individual_params/
>                      └─ individual_params/
>                         └─ config/
>                             ├─ default/
>     +                       └─ tutorial_vehicle/
>     +                           └─ tutorial_vehicle_sensor_kit_launch/
>     +                               ├─ imu_corrector.param.yaml
>     +                               ├─ sensor_kit_calibration.yaml
>     +                               └─ sensors_calibration.yaml
>     ```
> 

Reviewer: I believe this section is redundant and a little confusing for new users, I think it is better to only make reference to the `tutorial_vehicle_individual_params` repo and delete the stuff related to YOUR-VEHICLE-NAME for this part of the guide.
```suggestion
##Placing your `individual_parameters` repository inside Autoware

[Previously on this guide](../../creating-your-autoware-meta-repository/creating-autoware-meta-repository.md), we forked  the `autoware_individual_params` repository to create a [tutorial_vehicle_individual_params](https://github.com/leo-drive/tutorial_vehicle_individual_params) repository which will be used as an example for this section of the guide. 

Your individual_parameters repository should be placed inside your Autoware folder following the same folder structure as the one shown below:
??? note "sample folder structure for [`tutorial_vehicle_individual_params`](https://github.com/leo-drive/tutorial_vehicle_individual_params)"

    ```diff
    <YOUR-OWN-AUTOWARE-DIR>/
      └─ src/
           └─ param/
                └─ tutorial_vehicle_individual_params/
                     └─ individual_params/
                        └─ config/
                            ├─ default/
    +                       └─ tutorial_vehicle/
    +                           └─ tutorial_vehicle_sensor_kit_launch/
    +                               ├─ imu_corrector.param.yaml
    +                               ├─ sensor_kit_calibration.yaml
    +                               └─ sensors_calibration.yaml
    ```

```

Author: Yes, you are right. It can be confuser for new comers. This simplification is good for understandability. Done :+1:

> After that, we need to build individual_params package:

Reviewer: ```suggestion
After that, we need to build our `individual_params` package:
```

Author: done :+1:

```bash
colcon build --symlink-install --cmake-args -DCMAKE_BUILD_TYPE=Release --packages-up-to individual_params
```

> Now we are ready to use Autoware with vehicle_id as an arguments.
> For example, if our vehicle is more than one,
> the autoware_individual_params structure should be like this:

Reviewer: ```suggestion
Now you are ready to use Autoware with a vehicle_id as an argument. For example, if you are several, similar vehicles with different sensor calibration requirements, your autoware_individual_params structure should look like this:
```

Author: done :+1:
